client/views/add/add.js

```diff
@@ -9,6 +9,18 @@ function showModsError(reason) {
   currentError = Blaze.renderWithData(Template.form_error, reason, parentNode, nextNode);
 }
 
+function checkPrevMod(modBoxes) {
+  let modEmails = new Set();
+  for (let i = 0; i < modBoxes.length - 1; i++) {
```

Comment:
There's almost no need to write loops like this in Javascript anymore. They only create more opportunities for bugs. Instead, you can do something like this:
```javascript
const modEmails = new Set(modBoxes.map(b => b.value));
```

Pretty slick, right? That's probably confusing if you haven't seen that syntax before, so ask questions.